### DISCUSSION TITLE
How to Style the Calender widget?

### DISCUSSION THREAD

**Discussion by aslowwriter** [elkowar/eww#1263] [Q&A]
So I'm trying to style the calendar widget a bit more than just a global color change. For example I'd like the week numbers and the day names to be different colors to the rest of the days. I've dug a bit around in the GTK css documentation and found this: https://docs.gtk.org/gtk4/class.Calendar.html#css-nodes I've gotten the `calendar` and `calendar:selected` selectors to activate, even without applied classes which suggests to me that the other nodes described in the page should also be possible, but I cannot for the life of me figure out how to do it. Has anyone managed this?

**Comment by fidelski** [comment]
I am no expert but I now of the `calendar:indeterminate` selector (the non existing days) and there are the classes `button` and `highlight` applied. I think it's the `highlight` class you're looking for:
```css
calendar.highlight {
  color: blue;
}
```

**Comment by neyhden** [comment]
Had the same issue a while ago, issue is, that link is for GTK4, EWW uses GTK3 so it's not really documented, or I haven't found it.
Here's what I use for it: https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/gtk/theme/Adwaita/_common.scss#L3924
